docs/operations/mixed-workloads.md

```diff
@@ -0,0 +1,213 @@
+* `druid.query.scheduler.laning.maxLowPercent` – The maximum percent of query threads to handle low priority queries. The remaining query threads are dedicated to high priority queries.
+
+Consider also defining a [prioritization strategy](../configuration/index.md#prioritization-strategies) for the Broker to label queries as high or low priority. Otherwise, manually set the priority for incoming queries on the [query context](../querying/query-context.md).
+
+If you use a __manual laning strategy__, set the following:
+
+* `druid.query.scheduler.laning.lanes.{name}` – The limit for how many queries can run in the `name` lane. Define as many named lanes as needed.
+* `druid.query.scheduler.laning.isLimitPercent` – Whether to treat the lane limit as an exact number or a percent of the minimum of `druid.server.http.numThreads` or `druid.query.scheduler.numThreads`.
+
+With manual laning, incoming queries can be labeled with the desired lane in the `lane` parameter of the [query context](../querying/query-context.md).
+
+See [Query prioritization and laning](../configuration/index.md#query-prioritization-and-laning) for additional details on query laning configuration.
+
+### Example
+
+Example config for query laning with the high/low laning strategy:
+
+```
+# Laning strategy
+druid.query.scheduler.laning.strategy=hilo
+druid.query.scheduler.laning.maxLowPercent=20
+
+# Limit the number of HTTP threads for query processing
+# This value should be less than druid.server.http.numThreads
+druid.query.scheduler.numThreads=40
+```
+
+
+## Service tiering
+
+In service tiering, you define separate groups of Historicals and Brokers to manage queries based on the segments and resource requirements of the query.
+You can limit the resources that are set aside for certain types of queries.
+Many heavy queries involving complex subqueries or large result sets can hog resources away from high priority, interactive queries.
+Minimize the impact of these heavy queries by limiting them to a separate Broker tier.
+When all Brokers set aside for heavy queries are occupied, subsequent heavy queries must wait until the designated resources become available.
+A prolonged wait results in the later queries failing with a timeout error.
+
+Note that you can separate Historical processes into tiers without having separate Broker tiers.
+Historical-only tiering is not sufficient to meet the demands of mixed workloads on a Druid cluster.
+However, it is useful when you query certain segments more frequently than others, such as often analyzing the most recent data.
+Historical tiering assigns data from specific time intervals to specific tiers in order to support higher concurrency on hot data. 
+
+The examples below demonstrate two tiers—hot and cold—for both the Historicals and Brokers. The Brokers will serve short-running, light queries before long-running, heavy queries. Light queries will be routed to the hot tiers, and heavy queries will be routed to the cold tiers.
```

Comment:
I notice that we switch to hot and cold here. Are these related to "heavy" and "light" introduced above?

Reply:
Not for this PR, but I would like to see an example of how  two queries, one heavy/cold or light/hot and how they would be routed using this strategy. I think that might be a good result of the research of using the flame graphs, etc.

Reply:
In this doc I'm using "hot/cold" for data and tiers and "heavy/light" for describing the queries. The crucial bit that connects the two concepts is in L111: "Light queries will be routed to the hot tiers, and heavy queries will be routed to the cold tiers." Is that clear enough?

Reply:
Sounds good.